plugins/callback/foreman.py

```diff
@@ -19,6 +20,26 @@
       - whitelisting in configuration
       - requests (python library)
     options:
+      report_type:
+        description:
+          - endpoint type for reports: foreman or proxy
+        env:
+          - name: FOREMAN_REPORT_TYPE
+        required: True
+        default: foreman
+        ini:
+          - section: callback_foreman
+            key: report_type
+      fact_type:
+        description:
+          - endpoint type for facts: foreman (proxy setting is not yet supported)
```

Comment:
```suggestion
          - "endpoint type for facts: foreman (proxy setting is not yet supported)"
```

YAML is picky if you have colons in your strings

Reply:
On the other hand, given this is not yet supported, why introducing it here?

Reply:
True, removed.